packages/general/src/log/Logger.ts

```diff
@@ -130,11 +130,19 @@ export class Logger {
         Logger.logger.splice(index, 1);
     }
 
+    /**
+     * Check if a logger with the matching identifier exists.
+     * @param identifier The identifier of the logger
+     */
+    public static hasLoggerForIdentifier(identifier: string) {
+        return Logger.logger.some(logger => logger.logIdentifier === identifier);
+    }
+
     /**
      * Get the logger with the matching identifier.
      * @param identifier The identifier of the logger
      */
-    public static getLoggerforIdentifier(identifier: string) {
+    public static getLoggerForIdentifier(identifier: string) {
```

Comment:
Crappy gh didn't save my comment because I hit "approve" before "start a review", then crappy automerge (why?) merged but I was going to say that, since this is a breaking change already... ;)

Maybe we could move away from the c-like API for managing log destinations and get rid of the confusing overuse of the term "logger" (both the logging API and the logging back ends seem to be called loggers).

I'm thinking we could instead have an API that is `Logger.defaultDestination: Logger.Destination` and `Logger.destinations: Set<Logger>` instead.  That'd provide both mutability and introspection and get rid of the concept of identifiers altogether.